keyboards/kegen/gboy/readme.md

```diff
@@ -0,0 +1,28 @@
+# G-Boy
+
+![G-Boy](https://i.imgur.com/APwppRw.png)
```

Comment:
This uses a smaller, but still high res image, that is more appropriate for how github renders markdown. 
```suggestion
![G-Boy](https://i.imgur.com/APwppRwh.png)
```